src/main/java/org/mamute/dao/QuestionDAO.java

```diff
@@ -278,5 +277,126 @@ public Question questionWith(Attachment attachment) {
 				.setParameter("a", attachment)
 				.uniqueResult();
 	}
-}
 
+	public void delete(Question question) {
```

Comment:
@leocwolter, in the case of deleting a whole thread, should I also logically delete the Answers/Comments/Watchers/etc as well?